src/pybind/mgr/dashboard/settings.py

```diff
@@ -38,14 +38,23 @@ def has_default_value(name):
 class SettingsMeta(type):
     def __getattr__(cls, attr):
         default, stype = getattr(Options, attr)
-        return stype(mgr.get_config(attr, default))
+        if stype == bool and str(mgr.get_config(attr,
+                                                default)).lower() == 'false':
+            value = False
+        else:
+            value = stype(mgr.get_config(attr, default))
+        return value
 
     def __setattr__(cls, attr, value):
         if not attr.startswith('_') and hasattr(Options, attr):
             mgr.set_config(attr, str(value))
         else:
             setattr(SettingsMeta, attr, value)
 
+    def __delattr__(self, attr):
+        if not attr.startswith('_') and hasattr(Options, attr):
+            mgr.set_config(attr, None)
```

Comment:
Using `None` as "remove this setting" means that no one will be able to store `None` as a real settings value. Can we add a `mgr.remove_config()`?

Reply:
> Using None as "remove this setting" means that no one will be able to store None as a real settings value.

That's true, but for which type would you like to be able to have `None` as value? Neither `str` nor `int` nor `bool` requires that.

> Can we add a mgr.remove_config()?

I'm sadly not able to do that because of my lack of knowledge of C. But that knowledge would be required to implement it as there's no ready to use function. But I think what we have here is sufficient.